crates/bin/pcli/tests/network_integration.rs

```diff
@@ -39,9 +40,9 @@ const TIMEOUT_COMMAND_SECONDS: u64 = 20;
 // The "unbonding_delay" value is specified in blocks, and in the smoke tests,
 // block time is set to ~500ms, so we'll take the total number of blocks
 // that must elapse and sleep half that many seconds.
-static UNBONDING_DURATION: Lazy<Duration> = Lazy::new(|| {
+static UNBONDING_DELAY: Lazy<Duration> = Lazy::new(|| {
     let blocks: f64 = std::env::var("UNBONDING_DELAY")
-        .unwrap_or("100".to_string())
+        .unwrap_or("50".to_string())
```

Comment:
Adjusted default value to match what we being set in the smoke tests, via env var override.